_test_/users.test.js

```diff
@@ -0,0 +1,13 @@
+const supertest = require('supertest');
+const endpoint = 'https://rest-fetch.herokuapp.com/users'
```

Comment:
Should this run the test on the deployed API or the local API? This needs to be confirmed.

Reply:
@subhangi2731 Please change the link to a local API. We can merge it then